src/Repository/InvoiceRepository.php

```diff
@@ -39,11 +39,13 @@ public function findByDrivingSchoolId($drivingSchoolId): array
             ;
     }
 
-    public function findByInvoiceName(string $name): array
+    public function findByInvoiceNameAndDescription(string $name): array
     {
         return $this->createQueryBuilder('q')
             ->andWhere('q.name LIKE :name')
+            ->orWhere('q.description LIKE :description')
             ->setParameter('name', '%' . $name . '%')
+            ->setParameter('description', '%' . $name . '%')
```

Comment:
Watch out, you setted both parameter to name meanwhile you are looking for name or description

Reply:
name c'est juste le truc que tu écris en entrer je vais changer en search